src/Microsoft.Dnx.Runtime/Helpers/RoslynAnalyzerHelper.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.Versioning;
+using Microsoft.Dnx.Compilation;
+using NuGet;
+
+namespace Microsoft.Dnx.Runtime.Helpers
+{
+    internal static class RoslynAnalyzerHelper
+    {
+        private static readonly string AnalyzerBaseFolder = "analyzers";
+        private static readonly string AnalyzerLanguage = "cs";
```

Comment:
Yeah, we may need to look at the layering of this. I'd prefer to avoid having specific Roslyn knowledge in the runtime, but it might be tricky given that we really need to use the NuGet loader/dependency resolver.